pages/linux/rpm.md

```diff
@@ -30,3 +30,7 @@
 - Show changed, missing and/or incorrectly installed files of matching packages:
 
 `rpm --verify --all '{{php-*}}'`
+
+- Display changelog of a specific package:
+
+`rpm --query --changelog {{package_name}}`
```

Comment:
The other examples use `package_name.rpm`. Which one should we use?

Reply:
In this case `package_name` is the name of the package without `.rpm`.
It is the installed `rpm` that is on the system.
It is one of the packages of the output from:
`rpm --query --all`